javascript/ql/src/semmle/javascript/security/dataflow/ExceptionXss.qll

```diff
@@ -0,0 +1,86 @@
+/**
+ * Provides a taint-tracking configuration for reasoning about cross-site
+ * scripting vulnerabilities where the taint-flow passes through a thrown
+ * exception.
+ */
+
+import javascript
+
+module ExceptionXss {
+  import DomBasedXssCustomizations::DomBasedXss as DomBasedXssCustom
+  import ReflectedXssCustomizations::ReflectedXss as ReflectedXssCustom
+  import Xss::DomBasedXss as DomBasedXss
+  import Xss::ReflectedXss as ReflectedXSS
+  import Xss::StoredXss as StoredXss
+  import Xss as XSS
+  
+  /**
+   * Holds if `node` cannot cause an exception containing sensitive information to be thrown.
```

Comment:
How did you arrive at this whitelist?

Reply:
By looking at the current results that I have and removing the obvious false positives. 

After the query has been deployed to LGTM I can start looking for more general FP filters.